sdcm/cluster.py

```diff
@@ -2316,6 +2316,14 @@ def install_mgmt(self, package_url: Optional[str] = None) -> None:
             manager_yaml["tls_key_file"] = tls_key_file
             manager_yaml["prometheus"] = f":{self.parent_cluster.params.get('manager_prometheus_port')}"
 
+            # TODO: remove this restriction to update_frequency configuration after manager-3.3 release
+            # `config_cache` parameter was introduced in version 3.2.7-0.20240509, skip this for older versions.
+            version = self.remoter.run(
+                "sctool version | grep 'Client version:' | awk -F': ' '{print $2}'").stdout.strip()
+            # example of version returned - 3.2.3~0.20231019.5b0db89a
+            if packaging.version.parse(version) >= packaging.version.parse("3.2.7-0.20240509"):
```

Comment:
Better to use following approach:
```
ComparableScyllaVersion(version) > "3.2.7-0.20240509"
```

Because:
```
>>> version = "3.2"
>>> ComparableScyllaVersion(version) > "3.2.7-0.20240509"
False
>>> packaging.version.parse(version) >= packaging.version.parse("3.2.7-0.20240509")
True
```

And:
```
>>> type(packaging.version.parse(version)), type(packaging.version.parse("3.2.7-0.20240509"))
(<class 'packaging.version.Version'>, <class 'packaging.version.LegacyVersion'>)
```

If, the scylla-manager case requires different approach for version comparison than the scylla itself, then need to add it to the `sdcm.utils.version_utils` module.

Reply:
yep, make sense, the approach is reworked.